records/2023-Q1-roadmap.md

```diff
@@ -0,0 +1,175 @@
+[Scala Contributors forum](https://contributors.scala-lang.org/c/scala-center/25).
+
+To have more information about our _completed_ projects, please see the
+[quarterly activity reports]({% link records.md %}).
+
+## Roadmap for 2023 Q1
+{: .no_toc}
+
+The following sections present our plan for the current quarter. Every
+project description is followed by the concrete results we will deliver, and
+their expected outcome on the Scala community.
+
+* Table of Contents
+  {:toc}
+
+### Make Scala tooling seamless for newcomers, and provide robust foundations for the Scala ecosystem
+
+#### Polish sbt and Coursier
+
+Most of the Scala projects are built with sbt, which uses Coursier under the hood.
+The Coursier CLI is also the recommended way to get started in Scala. We want to make sure these tools
+work well for everyone. In collaboration with the current maintainers Eugene Yokota and Alexandre Archambault, we will
+address the most important issues that are currently opened. Read our roadmap for Coursier [here](https://contributors.scala-lang.org/t/bugfixes-and-seamless-installation-process-for-coursier/6052).
+
+#### TASTy-MiMa
+
+[TASTy-query](https://github.com/scalacenter/tasty-query) is a work-in-progress
+library to read TASTy files, explore them, and ask semantic questions about
+them. It is independent from the compiler, and can be used as a building
+block to implement tools performing static analyses of Scala programs. An
+example is TASTY-MiMa, an upcoming tool “like
+[MiMa](https://github.com/lightbend/mima), but for TASTy”.
+
+We plan to implement support for semantic queries such as “are two types
+equivalent?”, “is a type a subtype of another one?”, etc., and to publish a
+prototype of TASTy-MiMa.
+
+You can read the full roadmap
+[here](https://contributors.scala-lang.org/t/the-scala-centers-roadmap-for-tasty-query/5564).
+
+#### Create a Stable API for the Scala 3 Presentation Compiler
+
+Currently, Metals works only for a limited subset of Scala 3 versions. Furthermore, we need to publish
+a new release of Metals after every new release of the compiler to support it.
+
+We would like to create a stable API for the Scala 3 presentation compiler to untie Metals to the compiler
+release cycle and support a wider range of Scala 3 versions (including experimental versions of Scala 3).
+
+#### Reduce the Number of Ways to Import Scala Projects in IDEs
+
+Currently, when starting new projects in Metals the build server that is used by default is Bloop. Part of this is for historical reasons, but part of it is also due to the speed of Bloop and the integrations that exist for it. Bloop has been incredibly important in the Build Server arena. However, there is also BSP implementation in sbt and in Mill and even when a `.bsp/<build-tool>.json` entry exists, Metals will still force Bloop on the user unless they manually switch. This can cause confusion and also may add extra steps into choosing a build server that shouldn't be needed.
```

Comment:
```suggestion
Currently, when starting new projects in Metals the build server that is used by default is Bloop. Part of this is for historical reasons, but part of it is also due to the speed of Bloop and the integrations that exist for it. Bloop has been incredibly important in the Build Server arena. However, there are also BSP implementations in sbt and in Mill and even when a `.bsp/<build-tool>.json` entry exists, Metals will still force Bloop on the user unless they manually switch. This can cause confusion and also may add extra steps in choosing a build server that shouldn't be needed.
```